web/src/Target.jsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useState } from "react";
+import Popup from "./Popup";
+import { Button, Text } from "@patternfly/react-core";
+
+import NetworkManager from "./NetworkManager";
+
+export default function Ip() {
```

Comment:
NP: It does not matter which name you put in the default export, but we tend to use the same as chosen for the file. Maybe `IpAddresses` suits better for both, the component and the file. But it is just a suggestion. I really sucks at naming stuff :)

BTW,  did you know that `default` exports are not named export? So you can import them with the name of your preference, although usually we pick the original name. More about _exports_ and _imports_ at https://javascript.info/import-export#a-word-against-default-exports and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
Ip what? Target what?
I don't know the React naming conventions, but the words you chose are way too generic. And you have "resolved" David's note without answering :)